.bazelci/presubmit.yml

```diff
@@ -3,24 +3,12 @@ tasks:
   web_testing_ubuntu2004:
     name: "rules_webtesting test in ubuntu"
     platform: ubuntu2004
+    shell_commands:
+    - "sh ./setup.sh"
```

Comment:
I don't like the dangling `setup.sh` file in the root of the repo. It only applies to a very specific environment. I would either:

  1. Move it into the `.bazelci` folder and call it `setup_ubuntu.sh`
  2. Just inline it here. Use yaml blocks to make it tidy